File: IntelligentRecruitment_microservice/recruiter-service/src/main/java/com/fedicode/recruiterservice/Dto/RecruiterResponse.java
package com.fedicode.recruiterservice.Dto;

import com.fedicode.recruiterservice.Entity.Status;
import lombok.Builder;


import java.time.LocalDateTime;
@Builder
public record RecruiterResponse(
        int id,
        String email,
        String companyName,
        String firstName,
        String lastName,
        String phone,
        Status status,
        String profileImage,
        LocalDateTime createdAt) {}

File: IntelligentRecruitment-UI/src/app/models/Dto/Recruiter/RecruiterResponse.ts
import {Status} from './Status';

export interface RecruiterResponse{
  id:number;
  email:string;
  companyName:string;
  firstName:string;
  lastName:string;
  phone:string;
  createdAt:string;
  status:Status;
  profileImage: string | null;
}
